.eslintrc.json

```diff
@@ -24,6 +24,7 @@
         "brace-style": ["error", "1tbs"], // Requires one true brace style
         "no-empty-function": "error", // Require an empty function to at least have a comment explaining why
         "no-var": "error", // Discourages using `var` and recommends using `let` or `const` instead,
+        "react/prop-types": 0, // Ignores proptype declaration and validation
```

Comment:
I know how to resolve the errors to this rule. Let me know if you would like help with that.